### PR TITLE
Add basic input tests for registration form

### PR DESCRIPTION
**Pull request by rogerhutchings**
Adds some basic input tests to the registration form - I'll circle back to this later on @srallen

### CODE REVIEW

File: packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    it('should have a password field', function () {
+      const input = rendered.find('[name="password"]')
+      expect(input).to.have.lengthOf(1)
+      expect(input.attr('type')).to.equal('password')
+      expect(input.attr('required')).to.equal('required')
+      expect(input.prop('minlength')).to.equal('8')
+    })
+
+    it('should have a passwordConfirm field', function () {
+      const input = rendered.find('[name="password"]')
```
Comment (srallen) [inline packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: Wouldn't this be the same input from the above test. Could you find by id to be sure it's the one intended to be tested?

Comment (eatyourgreens) [reply packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: I think the name is wrong. It should be `passwordConfirm` for the confirmation input?

File: packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      expect(input.attr('type')).to.equal('password')
+      expect(input.attr('required')).to.equal('required')
+      expect(input.prop('minlength')).to.equal('8')
+    })
+
+    it('should have an email field', function () {
+      const input = rendered.find('[name="email"]')
+      expect(input).to.have.lengthOf(1)
+      expect(input.attr('type')).to.equal('email')
+      expect(input.attr('required')).to.equal('required')
```
Comment (eatyourgreens) [inline packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: Should there be an input to confirm the email address too?

File: packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import RegisterForm from './RegisterForm'
+
+let wrapper
+
+describe.only('Component > RegisterForm', function () {
```
Comment (eatyourgreens) [inline packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]:
```suggestion
describe('Component > RegisterForm', function () {
```

File: packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js (hunk 2 above)
Comment (rogerhutchings) [reply packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: Possibly, although there isn't one in the base branch yet

Comment (srallen) [reply packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: That would probably help cut down on people submitting emails with typos in them.

Comment (eatyourgreens) [reply packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: We could spec it here then add it to the form.

Comment (srallen) [reply packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.spec.js]: I'll merge and spec it in the base branch before I add it in.

**Pull request merged**